py/bitbox02/bitbox02/communication/bitbox_api_protocol.py

```diff
@@ -680,11 +680,14 @@ def reboot(
         return True
 
     @staticmethod
-    def get_info(transport: TransportLayer) -> Tuple[str, Platform, Union[BitBox02Edition], bool]:
-        """
-        Returns (version, platform, edition, unlocked).
-        This is useful to get the version of the firmware when a usb descriptor is not available
-        (via BitBoxBridge, etc.).
+    def get_info(
+            transport: TransportLayer
+    ) -> Tuple[str, Platform, Union[BitBox02Edition], bool, Optional[bool]]:
+        """
+        Returns (version, platform, edition, unlocked, initialized).
+        This is useful to get the version of the firmware or the device unlocked/initialized status
+        when a usb descriptor is not available (via BitBoxBridge, etc.). In case the firmware does not
+        respond with the initialized byte yet None is returned.
```

Comment:
The docstring could be more explicit: "The initialized status is supported from firmware v9.20.0 (it is `None` if not supported)"